apps/concept-catalog/pages/[[...catalogId]]/index.tsx

```diff
@@ -103,40 +135,45 @@ export const getServerSideProps: GetServerSideProps = async ({
 }) => {
   const session = await getServerSession(req, res, authOptions);
 
-  if (!session) {
-    return {
-      props: {},
-    };
-  }
+  if (session?.user) {
+    const {accessToken} = session.user;
+    const pageNumb = query.page || 1;
 
-  const {accessToken} = session.user;
-  let catalogId = '';
-  let searchConceptResponse: SearchConceptResponseProps;
+    const pageQuery = JSON.stringify({
+      query: '',
+      pagination: {page: Number(pageNumb) - 1, size: 5},
+    });
 
-  try {
-    catalogId = query.catalogId[0];
-  } catch (error) {
-    console.log(error);
-  }
+    let catalogId = '';
+    let searchConceptResponse: SearchConceptResponseProps;
 
-  if (catalogId) {
-    searchConceptResponse = await searchConceptsForCatalog(
-      catalogId,
-      accessToken
-    );
-  }
+    try {
+      catalogId = query.catalogId[0];
+    } catch (error) {
+      console.log(error);
```

Comment:
@NilsOveTen @jeffreiffers Pleier vi logge error-meldinger i frontend, og fanges det opp i GCP?

Reply:
Er en cloud-function som brukes av fdk-portal, tror ikke noen av de andre applikasjonene sender feilmeldinger til den

Reply:
Er det ikke bedre måter å gjøre det på i en nextjs-applikasjon?

Reply:
Hvis ikke så har dere eksempel på bruk i fdk-portal.
axios: https://github.com/Informasjonsforvaltning/fdk-portal/blob/aff67a037f4d1ba14bd6b76edb2c001ff2f4790e/src/services/logging/index.ts
logging av feil via axios: https://github.com/Informasjonsforvaltning/fdk-portal/blob/cad706d92743d2bd6fad895bd43860d499e0b1f8/src/components/with-dataset/redux/saga.ts#L28-L34

Reply:
@pooriamehregan

Reply:
@terjesyl Bra innspill! Kanskje loggingfunksjonaliteten kan tas i sin egen PR og denne PRen burde ikke blokkeres av den grunn?  Dette er fordi Hege har ikke skrevet den koden, og omganget av oppgaven kan være passe nok for å få sitt eget issue og PR. Da foreslår jeg å heller åpne et issue på det.

@NilsOveTen Hmmm, NextJS-dokumentasjonen foreslår en pakke som kan brukes [Pino](https://www.npmjs.com/package/pino) for å få litt finere output hvis det er det man ønsker. Men to be honest tror jeg det er best å ikke dra inn ny avhengighet (altså et ekstern bibiotek) når det samme resultatet kan oppnås med den innebygde funksjonaliteten (concole.log). Men vi kan sikkert sette opp loggingfunksjonalitet mot GCP.

Reply:
Det er uansett out-of-scope for denne pr'en. Du kan eventuelt lage et issue